install_config/configuring_pvc_protection.adoc

```diff
@@ -0,0 +1,59 @@
+[[install-config-configuring-pvc-protection]]
+= Configuring Persistent Volume Claim Protection
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+{product-title} can be configured to have the 
+xref:../architecture/additional_concepts/storage.adoc#pvcprotection[Persistent volume claim (PVC) protection] feature enabled.
+
+[NOTE]
+====
+PVC protection is an alpha feature and may change in a future release of {product-title}.
+====
```

Comment:
It might be good to add the description of why someone would want to do this from lines 117-124 in the file you've changed above.

Reply:
I added this information. Thanks.